apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -62,8 +63,33 @@ class PageTemplateModal extends Component {
 				return prev;
 			},
 			{}
-		)
-	);
+		);
+
+		// Remove templates that include a missing block
+		rtn = this.filterTemplatesWithMissingBlocks( rtn );
+
+		return rtn;
```

Comment:
```suggestion
		return this.filterTemplatesWithMissingBlocks( rtn );
```

I think we can simplify this and switch the variable to const.

Reply:
I fixed this as part of the other comment.